### PR TITLE
Add python script to convert IDR image to Zarr

### PR DESCRIPTION
**Pull request by manics**
```
$ python idr-image2zarr.py 4995041

Downloading IDR image ID: 4995041
[1488, 1488, 9, 3, 1]
Saving 4995041/000-000-000.npy
Saving 4995041/000-001-000.npy
Saving 4995041/000-002-000.npy
Saving 4995041/001-000-000.npy
Saving 4995041/001-001-000.npy
Saving 4995041/001-002-000.npy
Saving 4995041/002-000-000.npy
Saving 4995041/002-001-000.npy
Saving 4995041/002-002-000.npy
Saving 4995041/003-000-000.npy
Saving 4995041/003-001-000.npy
Saving 4995041/003-002-000.npy
Saving 4995041/004-000-000.npy
Saving 4995041/004-001-000.npy
Saving 4995041/004-002-000.npy
Saving 4995041/005-000-000.npy
Saving 4995041/005-001-000.npy
Saving 4995041/005-002-000.npy
Saving 4995041/006-000-000.npy
Saving 4995041/006-001-000.npy
Saving 4995041/006-002-000.npy
Saving 4995041/007-000-000.npy
Saving 4995041/007-001-000.npy
Saving 4995041/007-002-000.npy
Saving 4995041/008-000-000.npy
Saving 4995041/008-001-000.npy
Saving 4995041/008-002-000.npy
Converting 4995041/*.npy to 4995041.zarr
0
1
2
3
4
5
6
7
8
<zarr.core.Array (1488, 1488, 9, 3, 1) uint16>
```

### CODE REVIEW

File: examples/idr2zarr/idr-image2zarr.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+print('Converting {} to {}'.format(srcglob, output))
+files = sorted(glob(srcglob))
+lastd = re.match(
+    r'(?P<iid>\d+)/(?P<z>\d+)-(?P<c>\d+)-(?P<t>\d+).npy', files[-1]
+    ).groupdict()
+for k, v in lastd.items():
+    lastd[k] = int(v)
+
+first = np.load(files[0])
+chunksize = (first.shape[0], first.shape[0], 5, 1, 5)
```
Comment (will-moore) [inline examples/idr2zarr/idr-image2zarr.py:56]: should one of these be ```first.shape[1]```?

Comment (manics) [reply examples/idr2zarr/idr-image2zarr.py]: Oops, well spotted!

**Comment by joshmoore** [comment]
Few things TBD from the Friday naming discussion.

**Comment by manics** [comment]
Hopefully https://github.com/ome/omero-ms-zarr/pull/7 can replace this

**Pull request closed without merging**